### PR TITLE
Release Notes - Mendix Cloud 8 Jul 2020

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

File: content/releasenotes/developer-portal/deployment.md [hunk 1]
```diff
@@ -10,6 +10,15 @@ These release notes cover changes to [Mendix Cloud](/developerportal/deploy/mend
 
 ## 2020
 
+### July 8th, 2020
+
+#### Mendix Cloud Fixes
+
+* We resolved an issue where deactivated users were still shown in the [Node Permissions](/developerportal/deploy/node-permissions) tab of the Security page for an app. (Tickets 90744 95319, 97722, 98474, 99978, 100493, 101595, 103007, and 103549)
+* We resolved an issue where you could not change the role of the inbuild administrator account for apps deployed to the cloud. (Tickets 87013 and 87805)
```
Comment (feddovanede) [inline content/releasenotes/developer-portal/deployment.md:18]: I think it should be 87605 instead of 87805.

Comment (MarkvanMents) [reply content/releasenotes/developer-portal/deployment.md]: Wow @feddovanede . Do you remember all the ticket numbers?
It does look more like a 6 when I wear my glasses. Thanks for picking it up.

**Comment by MarkvanMents** [comment]
Release successful - notified via Slack.

File: content/releasenotes/developer-portal/deployment.md (hunk 1 above)
Comment (feddovanede) [reply content/releasenotes/developer-portal/deployment.md]: Sort of coincidence: I was checking the ticket numbers to see if the one for a particular customer was mentioned, and then I found it was incorrect.

**Pull request merged**